src/collapse/collapse.ts

```diff
@@ -15,6 +15,12 @@ interface GetVisibleIdsInput {
   edges: GraphEdge[];
 }
 
+interface GetExpandPathInput {
```

Comment:
This filename in this directory structure is not great. Lets call it something else like `collapseUtils` or something..